docs/src/docs/asciidoc/maven-plugin.adoc

```diff
@@ -133,7 +133,7 @@ Build Configuration]. It is also possible to customize the plugin within a
 [source,xml]
 ----
 <buildArgs>
-    <arg>--argument</arg>
+    <buildArg>--argument</buildArg>
```

Comment:
Did it change too for specifying JVM arguments? 
Currently we have:
```
<jvmArgs>
    <arg>argument1</arg>
    <arg>argument2</arg>
</jvmArgs>
```

Reply:
Yes, for tags mapped as collections (like `buildArgs`), the inner tags can use any name. The convention is to use the singular form of the plural tag name.

Reply:
Thank you both!